common/button/button-en.html

```diff
@@ -0,0 +1,428 @@
+---
+{
+	"title": "Buttons",
+	"language": "en",
+	"altLangPage": "button-fr.html",
+	"description": "Buttons page including working examples to test how various button styles appear and function",
+	"dateModified": "2023-06-14"
+}
+---
+
+<h2>On this page:</h2>
+
+<ul>
+	<li><a href="#purpose">Purpose</a></li>
+	<li><a href="#basic">Basic use</a></li>
+	<li><a href="#enhanced">Enhanced use</a></li>
+	<li><a href="#colours/styles">Colours/Styles</a></li>
+	<li><a href="#calltoaction">Call to Action</a></li>
+	<li><a href="#sizing">Sizing</a></li>
+	<li><a href="#fullwidth">Full Width</a></li>
+	<li><a href="#activestate">Active State</a></li>
+	<li><a href="#disabledstate">Disabled State</a></li>
+	<li><a href="#bghorizontal">Button Groups (Horizontal)</a></li>
+	<li><a href="#bgvertical">Button Groups (Vertical)</a></li>
+	<li><a href="#buttontoolbar">Button Toolbar</a></li>
+	<li><a href="#inputgroups">Input Groups</a></li>
+	<li><a href="#datatoggle">Data-toggle Buttons</a></li>
+</ul>
+
+<h2>Button vs links</h2>
+<dl class="dl-horizontal">
+<dt>Button</dt>
+<dd>A button will perform an action on the current page without moving the focus or without navigating the user away</dd>
+<dt>Link</dt>
+<dd>A link will send the user at a location, in the same page via an anchor (`#`) link or to an another page without sending any data over.</dd>
+</dl>
+<p>Do not use the role button on an anchor, use a button element instead.</p>
+
+<h2 id="basic">Basic use</h2>
+
+<h3>Default</h3>
```

Comment:
There is no h3 sibling under the h2 scope, Please merge this title with the h2 parent and bump up the all the sub heading

Reply:
Changes made in the commit "Updated working examples".